packages/babel-parser/src/plugins/flow.js

```diff
@@ -2455,6 +2456,7 @@ export default (superClass: Class<Parser>): Class<Parser> =>
         node.callee = base;
         node.typeArguments = this.flowParseTypeParameterInstantiation();
         this.expect(tt.parenL);
+        // $FlowFixMe
```

Comment:
Not great, but related to the [TODO](https://github.com/babel/babel/pull/8002/files#diff-b8d40ce8ecac1a4432171ce346567e3dR542) around converting arguments into `$ReadOnlyArray`, which requires quite a bit more changes.